Packs/Expanse/Integrations/Expanse/Expanse.py

```diff
@@ -914,6 +957,116 @@ def exposures_command():
     return_outputs(human_readable, ec, exposures)
 
 
+def domains_for_certificate_command():
+    """
+
+    :return:
+    """
+    search = demisto.args()['common_name']
+    params = {
+        "commonNameSearch": search
+    }
+    token = do_auth()
+
+    observed_ips = []
+    matching_domains = []
+
+    certificates = _fetch_certificates(params=params, token=token)
+    for certificate in certificates:
+        certificate_details = _fetch_certificate(md5_hash=certificate['certificate']['md5Hash'], token=token)
+        for ip in certificate_details['details']['recentIps']:
+            observed_ips.append(ip['ip'])
+
+    for ip in observed_ips:
+        params = {
+            'inetSearch': ip,
+            'assetType': 'DOMAIN'
+        }
+        matching_domains += _fetch_ips(params=params, token=token)
+
+    if len(matching_domains) == 0:
+        demisto.results("No data found")
+        return
+
+    context = get_expanse_certificate_to_domain_context(common_name=search, data=matching_domains)
+
+    ec = {
+        'Expanse.IPDomains(val.SearchTerm == obj.SearchTerm)': context
+    }
+
+    raw_domain_objects = context['DomainList']
+    del context['DomainList']  # Remove full objects from human readable response
+    human_readable = tableToMarkdown("Expanse Domains matching Certificate Common Name: {search}".format(search=search),
+                                     context)
+    context['DomainList'] = raw_domain_objects
+    return_outputs(human_readable, ec, matching_domains)
+
+
+def _fetch_certificates(params, token):
+    """
+    Fetches all certificates that match the provided params.
+
+    :param params: Search parameters
+    :param token: Expanse Refresh token
+    :return: List of certificate objects
+    """
+    certificates = []
+    results = http_request('GET', 'assets/certificates', params, token=token)
+    try:
+        if len(results['data']) > 0:
+            certificates += results['data']
+            next_page = results.get('pagination', {}).get('next', None)
+            while next_page is not None:
+                params['pageToken'] = get_page_token(next_page)
+                results = http_request('GET', 'assets/certificates', params, token=token)
+                certificates += results['data']
+                next_page = results.get('pagination', {}).get('next', None)
+        return certificates
+    except Exception:
+        demisto.results("No data found")
+        return []
+
+
+def _fetch_certificate(md5_hash, token):
+    """
+    Returns details for a single certificate.
+
+    :param md5_hash: Search term for certificates
+    :param token: Expanse Refresh token
+    :return: Certificate details objects
+    """
+    try:
+        return http_request('GET', 'assets/certificates/{}'.format(md5_hash), {}, token=token)
+    except Exception:
+        demisto.results("No data found")
```

Comment:
better to call `demisto.results` in the command func and here just return empty array